config/routes.rb

```diff
@@ -52,6 +52,8 @@
           get :info
         end
       end
+
+      get 'leaderboard_stats' => 'users#fetch_leader_board_stats'
```

Comment:
Since this is already a custom resource shall we resource-fy like:

```suggestion
      resource :leaderboard, only: show
```